### PR TITLE
Use GitHub actions

### PR DESCRIPTION
**Pull request by kohrVid**
Closes #16.

This PR replaces our Travis CI configuration with a Github Action workflow. The workflow has been set up to run rspec tests with the following rubies:

* 2.3
* 2.4
* 2.5
* 2.6
* 2.7

### CODE REVIEW

**Review by hammady — CHANGES_REQUESTED** [review]

File: Gemfile.lock [hunk 1]
```diff
@@ -12,21 +12,15 @@ GEM
...
     coderay (1.1.2)
-    coveralls (0.8.23)
-      json (>= 1.8, < 3)
-      simplecov (~> 0.16.1)
-      term-ansicolor (~> 1.3)
-      thor (>= 0.19.4, < 2.0)
-      tins (~> 1.6)
     diff-lcs (1.3)
-    docile (1.3.2)
+    docile (1.1.5)
```
Comment (hammady) [inline Gemfile.lock:16]: This was downgraded.

File: Gemfile.lock [hunk 2]
```diff
@@ -59,18 +53,12 @@ GEM
       rspec-support (~> 3.9.0)
     rspec-support (3.9.2)
     rubyzip (1.1.7)
-    simplecov (0.16.1)
-      docile (~> 1.1)
+    simplecov (0.14.1)
```
Comment (hammady) [inline Gemfile.lock:56]: Downgraded.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/rayyansys/rayyan-formats.svg?branch=master)](https://travis-ci.org/rayyansys/rayyan-formats)
+![Build Status](https://github.com/rayyansys/rayyan-formats/actions/workflows/ruby_2.3.yml/badge.svg)
```
Comment (hammady) [inline README.md:1]: Is this correct?

Comment (kohrVid) [reply README.md]: Sorry - that's from the old workflow I think. Have updated the PR with the correct URL

**Pull request merged**